libs/template/templates/default-python/template/{{.project_name}}/README.md.tmpl

```diff
@@ -2,18 +2,40 @@
 
 The '{{.project_name}}' project was generated by using the default-python template.
 
+For documentation on the Databricks asset bundles format use for this project,
+and for CI/CD configuration, see https://docs.databricks.com/aws/en/dev-tools/bundles.
+
 ## Getting started
 
-0. Install UV: https://docs.astral.sh/uv/getting-started/installation/
+Choose how you want to work on this project:
+
+(a) Directly in your Databricks workspace, see
+    https://docs.databricks.com/dev-tools/bundles/workspace.
+
+(b) Locally with an IDE like Cursor, VS Code, or PyCharm, see
+    https://docs.databricks.com/vscode-ext and https://www.databricks.com/blog/announcing-pycharm-integration-databricks.
```

Comment:
PyCharm doesn't have bundle integration.

Reply:
Oh I thought it was supposed to have parity? I'll remove this. Let's discuss offline.